src/pcapi/core/bookings/api.py

```diff
@@ -189,3 +189,10 @@ def update_confirmation_dates(
         booking.confirmationDate = compute_confirmation_date(beginning_datetime, booking.dateCreated)
     repository.save(*bookings)
     return bookings
+
+
+def cancel_expired_booking_by_system(booking: Booking) -> None:
```

Comment:
Puisqu'on parle d'une réservation expirée, est-ce que le suffixe "_by_system" est nécessaire ?

```suggestion
def cancel_expired_booking(booking: Booking) -> None:
```